Riot/ViewController/ReadReceiptsViewController.m

```diff
@@ -0,0 +1,157 @@
+/*
+ Copyright 2017 Aram Sargsyan
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ReadReceiptsViewController.h"
+#import <MatrixKit/MatrixKit.h>
+
+@interface ReadReceiptsViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic) MXRestClient* restClient;
+
+@property (nonatomic) NSArray <MXRoomMember *> *roomMembers;
+@property (nonatomic) NSArray <UIImage *> *placeholders;
+@property (nonatomic) NSArray <NSString *> *recieptDescriptions;
+
+@property (weak, nonatomic) IBOutlet UIView *overlayView;
+@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
+@property (weak, nonatomic) IBOutlet UIView *containerView;
+@property (weak, nonatomic) IBOutlet UITableView *receiptsTableView;
+
+@end
+
+@implementation ReadReceiptsViewController
+
+#pragma mark - Public
+
++ (void)openInViewController:(UIViewController *)viewController withRestClient:(MXRestClient *)restClient withRoomMembers:(NSArray <MXRoomMember *> *)roomMembers placeholders:(NSArray <UIImage *> *)placeholders receiptDescriptions:(NSArray <NSString *> *)receiptDescriptions
+{
+    ReadReceiptsViewController *receiptsController = [[[self class] alloc] initWithNibName:NSStringFromClass([self class]) bundle:nil];
+    receiptsController.restClient = restClient;
+    receiptsController.roomMembers = roomMembers;
+    receiptsController.placeholders = placeholders;
+    receiptsController.recieptDescriptions = receiptDescriptions;
+    
+    receiptsController.providesPresentationContextTransitionStyle = YES;
+    receiptsController.definesPresentationContext = YES;
+    receiptsController.modalPresentationStyle = UIModalPresentationOverFullScreen;
+    receiptsController.modalTransitionStyle = UIModalTransitionStyleCrossDissolve;
+    
+    [viewController presentViewController:receiptsController animated:YES completion:nil];
+}
+
+#pragma mark - Lifecycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    [self configureViews];
+    [self configureReceiptsTableView];
+    [self addOverlayViewGesture];
+}
```

Comment:
I suggest you to put code from this functions into `viewDidLoad` and separate code with comments. These functions are really short and it looks like an overhead to create function for 2 lines of code.